frontend/src/metabase-types/guards/common.ts

```diff
@@ -3,3 +3,39 @@ export const isObject = (
 ): value is Record<string | number | symbol, unknown> => {
   return typeof value === "object" && value !== null;
 };
+
+type ErrorWithMessage = {
```

Comment:
@kamilmielnik please check https://github.com/metabase/metabase/blob/3527c911d7556a068581c5240f2d6f6f2c338bb5/frontend/src/metabase/lib/errors/messages.ts#L3

Reply:
e217c685365691f8b46bfcb669bd30a0b84bc7be

Reply:
Assuming this change will go through: https://github.com/metabase/metabase/pull/57205#discussion_r2063329814
If not, I will update.